atomic_defi_design/qml/Exchange/Trade/SimpleView.qml

```diff
@@ -105,13 +111,10 @@ ColumnLayout
                     font.pixelSize: Style.textSizeSmall5
                     background: Rectangle { color: theme.backgroundColor }
                     validator: RegExpValidator { regExp: /(0|([1-9][0-9]*))(\.[0-9]{1,8})?/ }
-                    onTextChanged:
+                    onEditingFinished:
```

Comment:
The idea is good but it will ruin the user experience enormously to have to leave the field to see the update, better to have the update in real time, onEditFinished can be used to calculate best orders and fees by calling manually the backend function, what do you think ?